### PR TITLE
move /next to the main export, preserve older code in "/legacy"

### PR DESCRIPTION
**Pull request by tobowers**
We recently received a bug report on some code we don't really want people using, and our discord has been asking to move the "/next" export up a level to be the main export. This PR updates the exports, the tests, and the docs to reflect how "/next" is where we really want/expect people to start.

I've preserved the "/next" export in the package.json and added a new "/legacy" export in case anyone still needs code from there.

Semantic versioning "0.1.0" should prevent anyone from getting auto-updated to this latest version.

### CODE REVIEW

**Review by kafischer — APPROVED** [review]
nice!

**Comment by charlesvien** [comment]
LGTM

**Review by kafischer — CHANGES_REQUESTED** [review]
i think a few changes to the main readme would make sense here too

**Comment by tobowers** [comment]
@kafischer updated.

File: README.md [hunk 1]
```diff
@@ -47,22 +55,23 @@ We are solving problems all the way across the AI souls stack, including:
...
 ```
-/example-webapp
 /core
+/docs
+
 /integrations
   /discord_bots
   /telegram
-/docs
+/example-webapp
```
Comment (kafischer) [inline README.md:64]: I wonder if we should remove all the examples and integrations rn
From those directories in root

File: README.md [hunk 2]
```diff
@@ -47,22 +55,23 @@ We are solving problems all the way across the AI souls stack, including:
...
 ```
 
-- [`/example-webapp`](https://github.com/opensouls/socialagi-ex-webapp) contains an example integration of the socialagi library in a chat website 
 - [`/core`](./core) contains the library [`socialagi` NPM package source](https://www.npmjs.com/package/socialagi)
-- [`/integrations`](./integrations) contains examples of the library in action. Right now contains several stand-alone discord and telegram bot repos
 - [`/docs`](./docs) contains the documentation website for the project, running at [socialagi.dev](http://socialagi.dev)
+- (slightly outdated) [`/example-webapp`](https://github.com/opensouls/socialagi-ex-webapp) contains an example integration of the socialagi library in a chat website 
+- (slightly outdated) [`/integrations`](./integrations) contains examples of the library in action. Right now contains several stand-alone discord and telegram bot repos
```
Comment (kafischer) [inline README.md:70]: should we remove?

**Comment by tobowers** [comment]
@kafischer removed :)

**Review by kafischer — APPROVED** [review]

**Pull request merged**